pkg/transport/transport.go

```diff
@@ -84,6 +85,9 @@ func addRoute(e *echo.Echo, p string, s service.Service, logger log.Logger) {
 func addContextAwareRoute(e *echo.Echo, p string, w func(c echo.Context) service.Service, logger log.Logger) {
 	e.POST(p, func(c echo.Context) error {
 		s := w(c)
+		if s == nil { // error in request -> was already logged
+			return fmt.Errorf("invalid request")
+		}
```

Comment:
hm, shouldn't we let `ServiceGenerator` also let return an `error`? What do you think?

Reply:
Yes, it makes sense. serviceGenerator returns now an service and an error.

Error message is clearer now:
> {"caller":"main.go:239","err":"webhook validation failed for /_dynamicwebhook/: The webhook_url has an unexpected format 'https://foo'","ts":"2021-03-25T20:08:30.029349769+01:00"}